eng/scripts/Update-PkgVersion.ps1

```diff
@@ -0,0 +1,32 @@
+
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $PackageName,
+  [string] $NewVersionString,
+  [string] $ReleaseDate
+)
+
+. (Join-Path $PSScriptRoot ".." common scripts common.ps1)
+
+$pkgProperties = Get-PkgProperties -PackageName $PackageName
+$packageVersion = $pkgProperties.Version
+
+$packageSemVer = [AzureEngSemanticVersion]::new($packageVersion)
+
+if ([System.String]::IsNullOrEmpty($NewVersionString))
+{
+    $packageSemVer.IncrementAndSetToPrerelease()
+    & "${EngCommonScriptsDir}/Update-ChangeLog.ps1" -Version $packageSemVer.ToString() `
+    -ChangelogPath $pkgProperties.ChangeLogPath -Unreleased $True
+}
+else
+{
+    $packageSemVer = [AzureEngSemanticVersion]::new($NewVersionString)
+    & "${EngCommonScriptsDir}/Update-ChangeLog.ps1" -Version $packageSemVer.ToString() `
+    -ChangelogPath $pkgProperties.ChangeLogPath -Unreleased $False `
+    -ReplaceLatestEntryTitle $True -ReleaseDate $ReleaseDate
+}
+
+$versionFileContent = Get-Content -Path $pkgProperties.versionFile -Raw
```

Comment:
Do we think it is worth just calling Get-Version again here to get the version file location? Instead of adding it to the shared package properties type?

Reply:
I think it might be better to just append the versionFile to the properties. go is a special case where version is concerned. I still need to add a $SetupPythonConventions method in the `SemVer.ps1` to account for the `v` prefix that comes with go versions.